From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@LuisCabrer__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (luisca). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

dstrockis committed:
adding acs migration article

dstrockis committed:
ACS article for initial publish

dstrockis committed:
Including email address

cwatson-cat committed:
Added new portal help files for UI help pilot

ShawnJackson committed:
edit pass: tutorial-build-deploy-jenkins

ShawnJackson committed:
fixing merge conflict

ShawnJackson committed:
edit pass: tutorial-build-deploy-jenkins

ShawnJackson committed:
Update tutorial-build-deploy-jenkins.md

ShawnJackson committed:
Update tutorial-build-deploy-jenkins.md

daveirwin1 committed:
Added new distros (Ubuntu) and 'sudo' before commands

daveirwin1 committed:
Added line break, sudo

dstrockis committed:
Temporary article

daveirwin1 committed:
Added instructions for getting perf counters.

daveirwin1 committed:
Fixed line spacing

v-geberr committed:
add qs utterance back with toc

v-geberr committed:
fixed url to samples repo

v-geberr committed:
playing with DocFx tags

v-geberr committed:
changed name to range

v-geberr committed:
replaced range with highlight

v-geberr committed:
replaced highlight with full file

v-geberr committed:
js ref

v-geberr committed:
playing

v-geberr committed:
playing

v-geberr committed:
playing

v-geberr committed:
using range

v-geberr committed:
blue next button

v-geberr committed:
blue next button

Commit:
new

v-geberr committed:
fixing to code lines ranges

v-geberr committed:
code line correction for cmd line

v-geberr committed:
code line correction for cmd line 2

chetansriv committed:
User Story 1113429: SaaS App Tutorial: Apptio - Update

chetansriv committed:
User Story 1113429: SaaS App Tutorial: Apptio - Update

v-geberr committed:
cleanup

v-geberr committed:
acrolinx suggestions

dstrockis committed:
Fixing author

v-geberr committed:
explaining char mapping

chetansriv committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr

chetansriv committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr

chetansriv committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr

v-geberr committed:
Incorporated comments

chetansriv committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr

v-geberr committed:
applying color fix to all js code

cherylmc committed:
new

cherylmc committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into peerings

PatAltimore committed:
Add feedback; simplify scenario

v-geberr committed:
added link to code file on LUIS-Samples

Commit:
O16N correction

PatAltimore committed:
Add boxes to screenshots

v-geberr committed:
NOTE and quote S

cherylmc committed:
update

cherylmc committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into peerings

cherylmc committed:
update

cherylmc committed:
update

cherylmc committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into peerings

Commit:
classic

cherylmc committed:
update

alkohli committed:
procedure to get internal/private IP address of the device

Commit:
fiixng formatting

Commit:
pulling extra desc of output

rwike77 committed:
acrolinx

Commit:
classic portal deprecation

rwike77 committed:
more usability study updates

cherylmc committed:
update

cherylmc committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into device2

chetansriv committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

chetansriv committed:
Pushing Apptio tutorial changes

v-thepet committed:
fix links

Commit:
refetching files

Commit:
task 323: edits, add note for CAA records support

rboucher committed:
new PR as old one (27067) corrupted.

rboucher committed:
TOC entry and change filename

saurabhsensharma committed:
Update backup-azure-system-state.md

Updates for System State GA Release.

mimig1 committed:
Cosmos DB: Fixed mention of capacity units.

hning86 committed:
Update release-notes-sprint-1.md

saurabhsensharma committed:
Update backup-azure-restore-system-state.md

Updating internal link to modified section header.

saurabhsensharma committed:
Update tutorial-backup-windows-server-to-azure.md

System state GA Release updates

saurabhsensharma committed:
Update tutorial-backup-windows-server-to-azure.md

saurabhsensharma committed:
Update backup-azure-system-state.md

alkohli committed:
minor fix

carolinacmoravia committed:
C6853: Hardcoded EN locale in link

Hello, @MarkMcGeeAtAquent,
Reported link is hardcoded so it can't lead to target laguages for localized versions. Could you help to merge this PR, please?
Many thanks in advance.

v-thepet committed:
delete security-center-powerbi

neilpeterson committed:
updated upgrade versions

neilpeterson committed:
updates

hning86 committed:
Update release-notes-sprint-1.md

carolinacmoravia committed:
C6855: Hardcoded EN locale in link

Hello, @mimig1,
Reported link is hardcoded so it can't lead to target laguages for localized versions. Could you help to merge this PR, please?
Many thanks in advance.

Commit:
freshness05

sethmanheim committed:
Reviews and updates

rastala committed:
Update how-to-use-gpu.md

v-kents committed:
updated link to aks

PRMerger19 committed:
Merge pull request #27146 from cherylmc/device2

Device2

Lisaco88 committed:
Merge pull request #27161 from dksimpson/dean-323

task 323: edits, add note for CAA records support

douglaslMS committed:
Added new monitoring article for SQL Data Sync.

Lisaco88 committed:
Merge pull request #27196 from carolinacmoravia/patch-442

C6853: Hardcoded EN locale in link

PRMerger19 committed:
Merge pull request #27208 from MarkusVi/freshness05

freshness05

iainfoulds committed:
vmquickstartconsistency

PRMerger18 committed:
Merge pull request #27210 from rwike77/usability

Usability study updates

Commit:
quickfix02

sethmanheim committed:
Freshness review

PRMerger16 committed:
Merge pull request #27215 from rastala/master

Update how-to-use-gpu.md

alkohli committed:
removed the region doc

rwike77 committed:
moved listening port section

rwike77 committed:
moved listening port section

Commit:
Merge pull request #27213 from sethmanheim/wk11-8

Reviews and updates

PatAltimore committed:
Add feedback

PatAltimore committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into patricka-quickstarts-master

v-shils committed:
Merge pull request #27148 from chetansriv/apptionewupdate

Apptionewupdate

Lisaco88 committed:
Merge pull request #26941 from v-geberr/1103-qs-utterance

add qs utterance back with toc

PRMerger12 committed:
Merge pull request #27209 from carolinacmoravia/patch-445

C6855: Hardcoded EN locale in link

PRMerger18 committed:
Merge pull request #27185 from saurabhsensharma/patch-10

Update tutorial-backup-windows-server-to-azure.md

v-thepet committed:
acrolinx

Commit:
Merge pull request #27225 from sethmanheim/sb-wk

Freshness review

PRMerger18 committed:
Merge pull request #27183 from mimig1/patch-156

Cosmos DB: Fixed mention of capacity units.

itechedit committed:
Merge pull request #26548 from ShawnJackson/tutorial-build-deploy-jenkins

edit pass: tutorial-build-deploy-jenkins

dstrockis committed:
Merge branch 'master' of https://github.com/microsoft/azure-docs-pr into acs-migration

PRMerger14 committed:
Merge pull request #27224 from MarkusVi/quickfix02

quickfix02

dstrockis committed:
Merge branch 'acs-migration' of https://github.com/dstrockis/azure-docs-pr into acs-migration

Commit:
Add -h

rjagiewich committed:
Merge pull request #26983 from hning86/master

sprint 1 release notes.

tfitzmac committed:
added reserved resource names

carolinacmoravia committed:
C6885: Hardcoded EN locale in link

Hello, @neilpeterson,
Reported link is hardcoded so it can't lead to target laguages for localized versions. Could you help to merge this PR, please?
Many thanks in advance.

carolinacmoravia committed:
C6878: Hardcoded EN locale in link

Hello, @numberbycolors,
Reported link is hardcoded so it can't lead to target laguages for localized versions. Could you help to merge this PR, please?
Many thanks in advance.

rwike77 committed:
updated graphic

dstrockis committed:
Moving content to Azure AD release notes

rwike77 committed:
updated graphic

rjagiewich committed:
Merge pull request #27092 from cherylmc/peerings

Peerings

v-shils committed:
Merge pull request #27179 from saurabhsensharma/patch-8

Update backup-azure-system-state.md

mmacy committed:
[acr] service limits include

rboucher committed:
add Next Steps

rwike77 committed:
updated graphic

rwike77 committed:
formatting

rboucher committed:
add numbers back in

PRMerger15 committed:
Merge pull request #27238 from daveirwin1/master

Updates for 9.2 agent.

PRMerger15 committed:
Merge pull request #27233 from raymondlaghaeian/patch-14

Add -h

rwike77 committed:
added line about app port

rjagiewich committed:
Merge pull request #27150 from v-thepet/newfix

production bug: fix links

Commit:
Merge pull request #27206 from neilpeterson/aks-upgrade

updated upgrade versions

Commit:
Merge pull request #27237 from carolinacmoravia/patch-449

C6885: Hardcoded EN locale in link

spelluru committed:
fixed the link text

tynevi committed:
Merge pull request #26194 from cwatson-cat/10-25-portal-articles-computeupd

Added new portal help files for UI help pilot

dstrockis committed:
Attempting to add an anchor

mmacy committed:
[acr] service limits include

v-shils committed:
Merge pull request #27217 from v-kents/1121400

updated AKS link (bug 1121400)

rjagiewich committed:
Merge pull request #27101 from PatAltimore/patricka-quickstarts-master

Add feedback; simplify scenario

Commit:
Merge pull request #27223 from iainfoulds/vmquickstartconsistency

VM quickstart consistency

tfitzmac committed:
added copy on variables

rboucher committed:
small corrections

alkohli committed:
changes from PR approvers

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into pmclassic

Commit:
Merge pull request #27250 from spelluru/adfuntilfix

fixed the link text

rmca14 committed:
Merge pull request #27219 from douglaslMS/sync-add-monitoring2

Added new monitoring article for SQL Data Sync.

tfitzmac committed:
removed link

Commit:
Merge pull request #27253 from tfitzmac/1108copyvariables

added copy on variables

PRMerger10 committed:
Merge pull request #25947 from dstrockis/acs-migration

[Azure Active Directory] ACS Retirement Doc

Lisaco88 committed:
Merge pull request #27173 from rboucher/grafana2

new PR as old one (27067) corrupted.

raiye committed:
Update cloud-services-guestos-update-matrix.md

GitHubber17 committed:
Merge pull request #27134 from alkohli/SCAIP

procedure to get internal/private IP address of the device

GitHubber17 committed:
Merge pull request #27235 from tfitzmac/1108names

added reserved resource names

cynthn committed:
Update maintenance-notifications.md

Commit:
storage: remove java article

GitHubber17 committed:
Merge pull request #27244 from mmacy/acr-quota-include-move

[acr] service limits include

Commit:
Merge pull request #27262 from tfitzmac/1109managed

removed link

GitHubber17 committed:
Merge pull request #27246 from rwike77/dotnet-tutorial

Dotnet tutorial

PRMerger17 committed:
Merge pull request #27239 from carolinacmoravia/patch-450

C6878: Hardcoded EN locale in link

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

PRMerger12 committed:
Merge pull request #27128 from cynthn/pmclassic

Planned maintenance classic

GitHubber17 committed:
Merge pull request #27266 from georgewallace/revertjava

storage: remove java article

PRMerger16 committed:
Merge pull request #27263 from raiye/patch-36

Update cloud-services-guestos-update-matrix.md

v-alje committed:
Merge pull request #27267 from MicrosoftDocs/FromPublicRepo

From public repo
